test/LockedNORI.test.ts

```diff
@@ -448,7 +448,19 @@ describe('LockedNori', () => {
 
   describe('role access', () => {
     // it.todo('only the token granter role can deposit tokens')
+
     describe('roles', () => {
+      describe('DEFAULT_ADMIN_ROLE', () => {
```

Comment:
the initializer of this contract grants the `TOKEN_GRANTER_ROLE` to whoever deploys the contract, but I couldn't explicitly find in the various initializer calls that the deployer gets the `DEFAULT_ADMIN_ROLE`, which they need to be able to grant the token granter role to the market contract... So just making sure that happens, I could take this out